module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -115,7 +114,7 @@ public function getResourceTags()
      *
      * @return array
      *
-     * @deprecated Use TagServiceInterface::getListTags()
+     * @deprecated Use \VuFind\Db\Service\TagServiceInterface::getListTags()
```

Comment:
Since TagServiceInterface wasn't used anywhere else, I changed this to follow the convention in other comments.